content/about/index.md

```diff
@@ -31,3 +31,20 @@ Haiku is developed by a growing community of volunteer developers from all aroun
 
   * [Discover the Haiku Community](/community)
   * Check out [Haiku Contributors](https://github.com/haiku/haiku/graphs/contributors) at GitHub
+
+### Who are the maintainers in the About window?
+
+Active developers are considered "Maintainers" of the Haiku operating system.
+
+  * Anyone who has contributed directly to the haiku or haikuporter codebase within the last 24 months.
+  * Anyone who has contributed directly to haikuports 12 or more times in the last 12 months.
```

Comment:
1 contribution a month is not exactly "maintainer" ... how about we make it 32 contributions over the past 12 months? Then it's a nice power of 2.

Reply:
Also with commit rights.

Reply:
I'm strictly going off of what was discussed on the mailing list. What if someone makes 1 commit of 10,000 lines?

Reply:
Once we migrate to gerrit, the language becomes easier.  "Anyone with commit access, or who has submitted and had 16 patches accepted"?

Reply:
Any word on this? It's been a while (3+ years to be exact) since this was opened...